verification/domain/models/comment.py

```diff
@@ -0,0 +1,13 @@
+class Comment:
+    def __init__(self, comment, created_by, created_at):
+        self.comment = comment
```

Comment:
comment should have attribute username.

Reply:
Make this variable private. Make use of getter and setter.

Reply:
@prashantramangupta created_by is the attribute which will serve purpose of the reviewer.